### PR TITLE
Updates to PBR library

### PR DESCRIPTION
**Pull request by niklasharrysson**
- Using new naming convention for nodes and inputs with underscore separators for readability and consistency.
- Changed nodes mixbsdf, mixedf, scalebsdf, scaleedf, addbsdf, etc. into polymorphic nodes mix, multiply, add, to follow MaterialX standard.
- Added test cases for new add, mix, multiply nodes.
- Moved <standard_surface> node to examples section.

### CODE REVIEW

File: documents/Examples/BxDF/standard_surface.mtlx [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<materialx version="1.36">
+  <!--
+    Autodesk Standard Surface
+  -->
+  <nodedef name="ND_standard_surface_surfaceshader" node="standard_surface" type="surfaceshader" nodegroup="pbr"
```
Comment (niklasharrysson) [inline documents/Examples/BxDF/standard_surface.mtlx]: Moved from being a pbrlib member to an external example built from pbrlib nodes.

File: documents/Libraries/pbrlib/genglsl/mx_multiply_bsdfC.glsl [hunk 2]
```diff
@@ -0,0 +1,14 @@
+void mx_multiply_bsdfC_reflection(vec3 L, vec3 V, BSDF in1, vec3 in2, out BSDF result)
```
Comment (jstone-lucasfilm) [inline documents/Libraries/pbrlib/genglsl/mx_multiply_bsdfC.glsl]: What would you think of the alternate naming convention "mx_multiply_bsdf_color" and "mx_multiply_bsdf_float"?  It's a bit more verbose, but potentially clearer to a new reader.

File: documents/Examples/BxDF/standard_surface.mtlx (hunk 1 above)
Comment (jstone-lucasfilm) [reply documents/Examples/BxDF/standard_surface.mtlx]: Excellent!

File: documents/TestSuite/pbrlib/edf/add_edf.mtlx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="add_edf_test" type="">
```
Comment (jstone-lucasfilm) [inline documents/TestSuite/pbrlib/edf/add_edf.mtlx]: Would it be worthwhile to remove these empty type strings for clarity?

Comment (niklasharrysson) [reply documents/TestSuite/pbrlib/edf/add_edf.mtlx]: Good catch, I will clean this up.

File: documents/Libraries/pbrlib/genglsl/mx_multiply_bsdfC.glsl (hunk 2 above)
Comment (niklasharrysson) [reply documents/Libraries/pbrlib/genglsl/mx_multiply_bsdfC.glsl]: Sounds good, I'll make the change.

**Pull request merged**